src/main/java/org/apache/cassandra/sidecar/routes/sstableuploads/SSTableUploadHandler.java

```diff
@@ -127,7 +123,7 @@ public void handleInternal(RoutingContext context,
         if (!limiter.tryAcquire())
         {
             String message = String.format("Concurrent upload limit (%d) exceeded", limiter.limit());
-            componentMetrics.rateLimitedCalls.metric.mark();
+            componentMetrics.rateLimitedCalls.metric.setValue(1);
```

Comment:
The component metrics seem wrong or misleading. When I think of components I think of SSTable Components. https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java#L206  But you are really tracking by extension, which is different than component. I think we should either rename the metric, or if we really want to track SSTable components, we should properly determine the SSTable component.

Reply:
Addressed this. Switched to using SSTable component name.